easybuild/easyconfigs/l/libsigc++/libsigc++-3.4.0-GCCcore-11.3.0.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'ConfigureMake'
+
+name = 'libsigc++'
+version = '3.4.0'
```

Comment:
It looks like this library can live side-by-side with version 2. At least the library name is differentiated, if this also applies to the header files we should add a unique name (`libsigc++3`) to differentiate it.

Reply:
I suspect you might not have intended to include version 2 anymore here since i think you only wanted version 3.4.0